Makefile

```diff
@@ -1,4 +1,5 @@
-PYTHON?=python
+SHELL := /bin/bash
```

Comment:
I set up a new environment, Windows Subsystem Linux for Ubuntu 20.04 LTS. I found it necessary to add this line because otherwise by default `make` uses `/bin/sh` as per https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html which causes `make venv` to fail with:
```
source .venv/bin/activate && make setup
/bin/sh: 1: source: not found
make: *** [Makefile:7: venv] Error 127
```
I'm not sure if you prefer users pass in the shell; if so I can remove this line.